types/angular-q-extras/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "dtslint/dt.json",
```

Comment:
For new packages, it should be just `"extends": "dtslint/dt.json"` here

Reply:
This is not possible, the interface in "@types/angular" is named "IQService" thus the "interface-name" needs to be disabled.

I could re-enable "max-line-length" but the call will be really ugly

```ts
allSettled<T1, T2, T3, T4, T5, T6, T7, T8, T9, T10>(values: [
          T1 | IPromise<T1>, T2 | IPromise<T2>, 
          T3 | IPromise<T3>, T4 | IPromise<T4>, 
          T5 | IPromise<T5>, T6 | IPromise<T6>,
          T7 | IPromise<T7>, T8 | IPromise<T8>,
          T9 | IPromise<T9>, T10 | IPromise<T10>
         ]): IPromise<[
           PromiseValue<T1>, PromiseValue<T2>, 
           PromiseValue<T3>, PromiseValue<T4>, 
           PromiseValue<T5>, PromiseValue<T6>, 
           PromiseValue<T7>, PromiseValue<T8>, 
           PromiseValue<T9>, PromiseValue<T10>
         ]>;
```

Reply:
You can disable them in source code like: `// tslint:disable-next-line interface-name`

Reply:
ok

what about the lines lenghts ? do I disable it line by line or respect the rule ?

Reply:
@plantain-00 can I have your point of view please ? I would like to finish this quickly.

Reply:
It seems `interface-name` breaks just one line, so it can be `// tslint:disable-next-line interface-name`;
It seems `max-line-length` breaks line 22 to 34, so  it can be `// tslint:disable max-line-length` at line 22 and `tslint:enable max-line-length` at line 34

Reply:
@plantain-00 I tried that but "disable" keyword is not authorized, only "disable-next-line" is

Reply:
See this for example: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/jquery/test/longdesc-tests.ts#L1

Reply:
@plantain-00 done